main.go

```diff
@@ -47,6 +47,7 @@ func main() {
 	flag.BoolVar(&opCfg.EnableWebhook, "enable-webhook", operatorconfig.DefaultEnableWebhook, "Enable the validating webhook server. Defaults to true.")
 	flag.BoolVar(&opCfg.EnablePlatformDetection, "enable-platform-detection", operatorconfig.DefaultEnablePlatformDetection, "Enable the detection of the underlying platform. Defaults to true.")
 	flag.StringVar(&opCfg.HealthProbeBindAddress, "health-probe-bind-addr", operatorconfig.DefaultHealthProbeAddr, "The address the health endpoint binds to.")
+	flag.IntVar(&opCfg.RequeuePeriodSeconds, "requeue-period", operatorconfig.DefaultRequeuePeriodSeconds, "Requeue period for a failed reconciliation (in seconds).")
```

Comment:
Potentially this could be different for different resources/controllers, adding it here may not be future-proof yet.

Reply:
Yes, it's not 100% future proof I agree. However the main controller is the one which reconciles the ExternalDNS resource, the others may even be removed in the future as they are not that useful now, when the operands are deployed in the operator's namespace.